data/human/hails.txt

```diff
@@ -1319,6 +1319,8 @@ phrase "friendly civilian"
 		"do secret deals with pirates."
 		"are aiding the Alphas."
 		"are secretly taking out Alpha enclaves."
+		"got the secret to antigravity repulsors from teddybears with antique guns."
```

Comment:
I feel like this one is a little bit too silly. :) Conspiracy theories and conspiracy theorists aren't funny; they're dangerous.

Reply:
I think this is not a conspiracy theory, but a subtle nod to Star Wars - remember Ewoks on Endor? They used bows, guns and some sort of a float thingy...

Reply:
It actually was a subtle nod to "The Road Not Taken (short story)" by Harry Turtledove.
But I'm okay with removing this.

Reply:
Removed.